scripts/download/wsl.mjs

```diff
@@ -8,7 +8,7 @@ import path from 'path';
 import { download } from '../lib/download.mjs';
 
 export default async function main() {
-  const v = '0.12';
+  const v = '0.13';
```

Comment:
```suggestion
  const v = '0.14';
```

Assuming #1429 will get merged before this PR means we will be at `v0.12.1`, which includes a `nerdctl` bump to 0.16.1 for both the Lima Alpine image and the WSL distro.

Switching the WSL distro to `v0.13` afterwards would revert `nerdctl` for WSL to 0.16.0 while keeping it for the Alpine ISO at 0.16.1.